core/src/main/scala/co.blocke.scalajack/BijectiveFunction.scala

```diff
@@ -10,6 +10,7 @@ object BijectiveFunction {
     implicit final class FunctionReverse[A, B](private val apply: A ⇒ B) extends AnyVal {
       @inline def reversedBy(unapply: B ⇒ A): BijectiveFunction[A, B] = BijectiveFunction(apply, unapply)
       def ⇄(unapply: B ⇒ A): BijectiveFunction[A, B] = reversedBy(unapply)
+      def =/=(unapply: B ⇒ A): BijectiveFunction[A, B] = reversedBy(unapply) // For those who don't like Unicode
```

Comment:
Yeah--I just needed something to approximate the unicode char which not everyone will have handy.  Open to alternatives.